apps/fabric-website/src/pages/Styles/IconsPage/IconsPage.tsx

```diff
@@ -154,11 +168,11 @@ ReactDOM.render(<MyIconButton />, document.body.firstChild);
           <h3>Fabric Icons tool</h3>
 
           <p>
-            The Fabric Icons tool, <a href="https://aka.ms/uifabric-icons">https://aka.ms/uifabric-icons</a>, lets you search and browse all
-            of Fabric's icons. You can also use it to create and maintain subsets of the icon font to use in your web apps, which are
+            The Fabric Icons tool, <a href="https://aka.ms/uifabric-icons">{'https://aka.ms/uifabric-icons'}</a>, lets you search and browse
```

Comment:
The changes to these links aren't related to the PR, but were necessary to commit the file due to a Prettier bug: https://github.com/prettier/prettier/issues/2347